panel/README.md

```diff
@@ -0,0 +1,29 @@
+# Panel
+
+## Usage
+
+The Panel component can be used in both XML and Kotlin
+
+Example of a padded panel in XML
+
+```xml
+<net.skyscanner.backpack.panel.bpkpanel
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    app:bpk_padding="true">
```

Comment:
is it still `bpk_padding` or just `padding`?

Reply:
🤔 if memory serves me well I stole the snippet from the demo app, I'll double check, thanks!

Reply:
I remember it was like this initially because we were not sure if just `padding` would clash with something. But the last thing I remember is that we were not using the bpk prefix anymore. @amolgupta would know better

Reply:
yes. We need to drop the `bpk_` prefix. A PR after moving things to single modules would be good.

Reply:
so just to be sure, at the minute the `bpk` is required, correct?

Reply:
I assume this PR would be merge only after moving to single module. A git rebase and rename variable would do the job.